third_party/CMakeLists.txt

```diff
@@ -13,7 +13,24 @@
 # limitations under the License.
 
 set(ABSL_PROPAGATE_CXX_STD ON)
+set(HAVE_OFF64_T OFF)
+set(ABSL_ENABLE_INSTALL ON)
```

Comment:
This change seems that it could be simplified.
Specifically say, you could add gRPC without installation nor protobuf.
Note that gRPC is payload-agnostic and we use flatbuffers only.